x/erc20/client/cli/tx.go

```diff
@@ -40,7 +40,7 @@ func NewTxCmd() *cobra.Command {
 // NewConvertCoinCmd returns a CLI command handler for converting a Cosmos coin
 func NewConvertCoinCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "convert-coin [coin] [receiver_hex]",
+		Use:   "convert-coin COIN [RECEIVER_HEX]",
```

Comment:
how come you are using both versions for arguments with and without `[ ]`?

Reply:
`[ ]` are for optional items according to the style guide [here](https://developers.google.com/style/code-syntax)